Cargo.lock

```diff
@@ -0,0 +1,3205 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I didn't mean to check this in. I will take it out. As a library, I don't believe this should be checked in.